src/community/infix-long-scale.ts

```diff
@@ -0,0 +1,39 @@
+import { AbstractInfixNotation } from "./infix-abstract";
+import type Decimal from "break_infinity.js";
+import { toSubscript, abbreviate } from "../utils";
+
+// Name comes from https://en.wikipedia.org/wiki/Long_and_short_scales
+export class InfixLongScaleNotation extends AbstractInfixNotation {
+  public readonly name: string = "Infix long scale";
+
+  protected groupDigits = 6;
```

Comment:
Codacy found an issue: [expected member-variable-declaration: 'groupDigits' to have a typedef](https://app.codacy.com/gh/antimatter-dimensions/notations/pullRequest?prid=6218257)